tests/test_samples.py

```diff
@@ -1380,6 +1366,23 @@ def test_samples_plot_labels():
         assert samples.get_label(col) == ax.get_xlabel()
 
 
+@pytest.mark.parametrize('kind', ['kde', 'hist', 'fastkde']
+                         if 'fastkde' in sys.modules else
+                         ['kde', 'hist'])
```

Comment:
Is this the best way to deal with fastkde, or should it be skipped/xfailed?

Reply:
This achieves the same as skipping. Skipping would appear in the tests as a little `s` rather than in the form of fewer tests. Would we prefer that?

Reply:
Fewer lines in the code is good. Being explicit that tests are skipped when not everything is installed is better. Mentioned in #110.